app/konnect/dev-portal/access-and-approval/manage-teams.md

```diff
@@ -0,0 +1,192 @@
+
+{% navtabs %}
+{% navtab Konnect UI %}
+In {% konnect_icon runtimes %} [**Runtime Manager**](https://cloud.konghq.com/us/runtime-manager), select a runtime group and follow these steps:
+
+1. Select **Gateway Services** from the side navigation bar, then **New Gateway Service**.
+
+1. From the **Add a Gateway Service** dialog, enter the following to create a new service:
+    * **Name:** `classic_pizzas`
+    * **Upstream URL:** `http://mockbin.org`
+    * Use the defaults for the remaining fields.
+1. Click **Save**. 
+1. To create an API product for your service, navigate to **API Products** in the sidebar, click **Add API Product** and enter `Pizza Ordering` in the **Product Name** field.
+1. Click **Publish to portal** from the **Actions** dropdown menu. This will allow developers to see your API product once you assign them to a team.
+1. Click **New Version** from the **Actions** dropdown menu and enter `v1` in the **Version Name** field and then click **Create**.
+1. Click the **v1** version and then click **Link** to add a service.
+1. Select a runtime from the **Select Runtime Group** dropdown menu, select `classic_pizzas` from the **Gateway Service** dropdown menu, and then click **Save**.
+1. Click the **Edit** icon next to the status of your product version and select **Published** from the **Edit version status** dialog and then click **Save**.
+
+{% endnavtab %}
+{% navtab API %} 
+{:.note}
+> **Note:** Make sure to include headers with the personal access token in your requests.
+
+These instructions only use the required fields in the following API specs:
+* [Create a service](https://developer.konghq.com/spec/3c38bff8-3b7b-4323-8e2e-690d35ef97e0/16adcd15-493a-49b2-ad53-8c73891e29bf#/Services/create-service)
+* [Create an API product](https://developer.konghq.com/spec/d420333f-01b0-464e-a87a-97acc92c2026/941af975-8cfa-40f7-afea-e82d248489a0#/API%20Products/create-api-product)
+* [Create an API product version](https://developer.konghq.com/spec/d420333f-01b0-464e-a87a-97acc92c2026/941af975-8cfa-40f7-afea-e82d248489a0#/API%20Product%20Version%20Specification/create-api-product-version-spec)
+
+If you want to use additional fields, see the API spec documentation.
+
+1. Create the `classic_pizzas` service:
+```bash
+curl --request POST \
+  --url https://<region>.api.konghq.com/v2/{runtime-group-id}/core-entities/services \
+  --data "name": "classic_pizzas" \
+  --data "host": "http://mockbin.org" \
+  --data "path": "/classic_pizzas" 
+```
+1. Create the `Pizza Ordering` API product:
+```bash
+curl --request POST \
+  --url https://<region>.api.konghq.com/v2/api-products \
+  --data "name": "Pizza Ordering" \
+  --data "description": "API product for pizza ordering" 
+```
+  Save the `portal-id` value from the output. This will be used later.
```

Comment:
```suggestion
```

I tested this 

```
{
  "labels": {},
  "id": "3f8c1951-a400-497e-a9ff-8bb702cf0aec",
  "name": "Pizza ordering",
  "description": "API product for pizza ordering",
  "portal_ids": [],
  "created_at": "2023-08-22T20:01:26.736Z",
  "updated_at": "2023-08-22T20:01:26.736Z"
}
```
Portal-id seems to return an empty array, so there may be nothing to save here.